autopilot/migrator/migrator.go

```diff
@@ -324,6 +324,16 @@ OUTER:
 
 		// return if there are no slabs to migrate
 		if len(toMigrate) == 0 {
+			res, err := m.alerts.Alerts(ctx, alerts.AlertsOpts{Offset: 0, Limit: -1})
```

Comment:
You could store the migration alert IDs on the `Migrator` and dismiss them as necessary instead. Not sure how worth it that is as a refactor at this point though.

Reply:
I considered that but didn't want to add any state to the migrator at this point. I don't mind refactoring it though.